<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCropper</class>
 <widget class="QDialog" name="ImageCropper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 55, 65);
background-color: qlineargradient(spread:reflect, x1:0, y1:0.483, x2:0.494318, y2:0.488, stop:0 rgba(0, 0, 0, 255), stop:0.272727 rgba(31, 46, 65, 255));
background-color: qlineargradient(spread:pad, x1:0.29, y1:0.795455, x2:0.880682, y2:0.773, stop:0 rgba(92, 98, 111, 255), stop:1 rgba(70, 75, 85, 255));;
background-color: qlineargradient(spread:pad, x1:0.1025, y1:0.21, x2:0.773, y2:0.982955, stop:0 rgba(64, 75, 85, 255), stop:1 rgba(47, 49, 52, 255));
background-color: qlineargradient(spread:pad, x1:0.228, y1:0.414773, x2:0.812182, y2:0.67, stop:0 rgba(50, 70, 96, 255), stop:1 rgba(32, 47, 66, 255));
border: 2px solid rgb(10,15,20);</string>
  </property>
  <widget class="QWidget" name="UpperFrame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.477273, y1:0.074, x2:0.478, y2:1, stop:0.00568182 rgba(40, 80, 111, 240), stop:1 rgba(27, 40, 56, 255));

border-top: 0px;
border-left: 0px;
border-right: 0px;
border-bottom: 1px;
border-style: solid;
border: 1px solid rgb(19, 28, 38);
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
border: none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Crop the image</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Close_button">
    <property name="geometry">
     <rect>
      <x>1175</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: none;
	border: 0px; 
}
QPushButton:hover
{ 
	background-color: rgb(249, 22, 34);
 }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>1161</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(93, 109, 122);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Preview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>167</width>
     <height>167</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-color: rgb(93, 109, 122);
border: 1px solid  rgb(10,15,20);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: transparent;
color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>Preview:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>240</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 rgb(0, 200, 250), 
				stop: 0.4 rgb(30, 150, 255),
                stop:1 rgb(60, 90, 255));
	border: none;
	border-radius: 3px;
}
QPushButton:hover
{
	
	background-color: rgb(25, 206, 255);
}</string>
   </property>
   <property name="text">
    <string>Apply current preview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: transparent;
border: none;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Click Left Mouse Button on the image and move mouse to crop the rectangle
and display it in the preview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: transparent;
border: none;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Click Left Mouse Button on the cropped rectangle and move mouse to move it</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: transparent;
border: none;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Press 'Apply current preview' to set current preview image as your new avatar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
